internal/pkg/service/buffer/store/receiver.go

```diff
@@ -133,6 +133,8 @@ func (s *Store) DeleteReceiver(ctx context.Context, receiverKey key.ReceiverKey)
 	_, err = op.MergeToTxn(
 		s.deleteReceiverBaseOp(ctx, receiverKey),
 		s.deleteExportBaseListOp(ctx, receiverKey),
+		s.schema.Configs().Mappings().InReceiver(receiverKey).DeleteAll(),
+		s.schema.Secrets().Tokens().InReceiver(receiverKey).DeleteAll(),
```

Comment:
Tu myslim chyba mazanie nejakych objektov.

Reply:
Dopises prosim na to test?

Reply:
Mám to připravený nad tímhle PR, mám to hodit sem?

Reply:
ako chces, mozes si to mergnut k sebe, alebo zavriet tvoj PR a pokracovat tu, alebo vytvorit novy PR, je mi to uplne jedno